test/CreateHandler/CreateHandlerCommandTest.php

```diff
@@ -43,6 +54,32 @@ private function reflectExecuteMethod()
         return $r;
     }
 
+    private function mockApplication()
+    {
+        $helperSet = $this->prophesize(HelperSet::class)->reveal();
+
+        $factoryCommand = $this->prophesize(Command::class);
+        $factoryCommand
+            ->run(
+                Argument::that(function ($input) {
+                    Assert::assertInstanceOf(ArrayInput::class, $input);
+                    Assert::assertContains('factory:create', (string) $input);
+                    Assert::assertContains('Foo\TestHandler', (string) $input);
+                    return $input;
+                }),
+                $this->output->reveal()
+            )
+            ->willReturn(0);
+
+        $application = $this->prophesize(Application::class);
+        $application->getHelperSet()->willReturn($helperSet);
+        $application->find('factory:create')->will([$factoryCommand, 'reveal']);
+
+        return $application;
+
+        $this->command->setApplication($application->reveal());
```

Comment:
unreachable statement

Reply:
Oh! That's not good! I'll remove the `return` statement!

Reply:
Actually, need to remove the `setApplication()` statements, as they happen in the test cases that call this method.